src/components/external-source/ExternalSourceManager.svelte

```diff
@@ -0,0 +1,897 @@
+              Owner
+              <input class="st-input w-100" disabled={true} name="owner" value={selectedSource.owner} />
+            </Input>
+
+            <Input layout="inline">
+              {`Start Time (${$plugins.time.primary.label})`}
+              <DatePicker
+                dateString={formatDate(new Date(selectedSource.start_time), $plugins.time.primary.format)}
+                disabled={true}
+                name="start-time"
+              />
+            </Input>
+
+            <Input layout="inline">
+              {`End Time (${$plugins.time.primary.label})`}
+              <DatePicker
+                dateString={formatDate(new Date(selectedSource.end_time), $plugins.time.primary.format)}
+                disabled={true}
+                name="end-time"
+              />
+            </Input>
+
+            <Input layout="inline">
+              {`Valid At (${$plugins.time.primary.label})`}
+              <DatePicker
+                dateString={formatDate(new Date(selectedSource.valid_at), $plugins.time.primary.format)}
+                disabled={true}
+                name="valid-at"
+              />
+            </Input>
+
+            <Input layout="inline">
+              {`Created At (${$plugins.time.primary.label})`}
+              <DatePicker
+                dateString={formatDate(new Date(selectedSource.created_at), $plugins.time.primary.format)}
+                disabled={true}
+                name="valid-at"
+              />
+            </Input>
+
+            <Collapse defaultExpanded={false} title="Event Types" tooltipContent="View Contained Event Types">
+              {#await effects.getExternalEventTypesBySource(selectedSource.key, selectedSource.derivation_group_name, user)}
+                <i>Loading...</i>
+              {:then eventTypes}
+                {#each eventTypes as eventType}
+                  <i>{eventType}</i>
+                {/each}
+              {/await}
+            </Collapse>
+            <Collapse defaultExpanded={false} title="Metadata" tooltipContent="View Event Source Metadata">
+              {#await effects.getExternalSourceMetadata(selectedSource.key, selectedSource.derivation_group_name, user)}
+                <em>loading metadata...</em>
+              {:then metadata}
+                {#if Object.keys(metadata).length}
+                  <Properties
+                    formProperties={Object.entries(metadata).map(metadataEntry => {
+                      return { name: metadataEntry[0], value: metadataEntry[1] };
+                    })}
+                  />
+                {:else if $getExternalSourceMetadataError}
+                  <em>Failed to retrieve External Source metadata.</em>
+                {:else}
+                  <em>Source has no metadata.</em>
+                {/if}
+              {:catch error}
+                <em>error loading metadata...try refreshing the page.</em>
+                {catchError(error)}
```

Comment:
The `effects.getExternalSourceMetadata` should already handle this `catchError` theoretically. This can be removed.